kong/runloop/wasm.lua

```diff
@@ -682,6 +683,155 @@ local function disable(reason)
 end
 
 
+local function register_property_handlers()
+  properties.reset()
+
+  properties.add_getter("kong.client.protocol", function(kong)
+    return true, kong.client.get_protocol(), true
+  end)
+
+  properties.add_getter("kong.nginx.subsystem", function(kong)
+    return true, kong.nginx.get_subsystem(), true
+  end)
+
+  properties.add_getter("kong.node.id", function(kong)
+    return true, kong.node.get_id(), true
+  end)
+
+  properties.add_getter("kong.node.memory_stats", function(kong)
```

Comment:
I'm a little on-the-fence about this one. Did it come up in your discovery of existing Golang plugins, or did you just add everything from `kong.node.*` PDK?

Reply:
The latter. The idea is that unless there's a technical limitation preventing it, everything in the PDK that is a `kong.x.get_y` getter should be available as a `kong.x.y` property. In general, I did not want to pick-and-choose, or redesign any interfaces.

Reply:
Mmmk, don't wanna hold this up on what is/isn't included then (we can always revise later on).